app/helpers/interactive_run_helper.rb

```diff
@@ -51,6 +51,12 @@ def interactive_data_div(interactive,run)
     data['interactive-name'] = interactive.name
     data['linked-interactives'] = interactive.linked_interactives_list.to_json
 
+    # default to normal size unless set in activity
+    data['font-size'] = 'normal'
+    if run && run.activity && run.activity.font_size
+      data['font-size'] = run.activity.font_size
+    end
+
```

Comment:
Is this used anymore, without the lara runtime?

Reply:
It isn't, but I didn't want to introduce a bunch of noise into the PR with deleting old LARA runtime files - that should be another (large) PR in the future.